### PR TITLE
Adds Akka Projections introduction video

### PR DESCRIPTION
**Pull request by renatocaval**
This builds on top of #743. I will rebase one #743 is merged.

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]

File: blog/_posts/2020-09-10-akka-projection-intro-video.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+author: Renato Cavalcanti
+short: This video introduces Akka Projections, a new Akka module to process a stream of events or records from a source to a projected model or external system.
+category: news
+tags: [videos]
+youtubeId: 0toyKxomdwo
+---
+
+{% include youtubePlayer.html id=page.youtubeId %}
+
+This video introduces [Akka Projections](https://doc.akka.io/docs/akka-projection/current/), a new Akka module to process a stream of events or records from a source to a projected model or external system.
```
Comment (patriknw) [inline blog/_posts/2020-09-10-akka-projection-intro-video.md:13]: btw, should we mention something about CQRS here? Could be good for google?

Comment (renatocaval) [reply blog/_posts/2020-09-10-akka-projection-intro-video.md:13]: Yep, I will add something

**Comment by renatocaval** [comment]
For some reason GitHub is not picking the latest change I did on this branch, so I'm opening another PR.

**Pull request closed without merging**